contrib/epee/include/serialization/keyvalue_serialization.h

```diff
@@ -73,19 +73,16 @@ public: \
 #define KV_SERIALIZE_N(varialble, val_name) \
   epee::serialization::selector<is_store>::serialize(this_ref.varialble, stg, hparent_section, val_name);
 
-  template<typename T> inline void serialize_default(const T &t, T v) { }
-  template<typename T> inline void serialize_default(T &t, T &&v) { t = std::forward<T>(v); }
```

Comment:
This code was actually kind of wrong before: it was meant to accept the second argument as universal reference, but isn't -- and so when you pass a non-rvalue into the second argument the second overload can't work.  I think changing it to this will solve the issue:
```C++
template <typename T, typename S>
inline void serialize_default(T &t, S &&v) { t = std::forward<S>(v); }
```
which is really what it was meant to be doing (accepting the second argument as universal reference).

Reply:
this did the trick